packages/documentation/src/sidebar.js

```diff
@@ -70,41 +65,8 @@ module.exports = {
               name: 'Updating Formation',
               // href: '/getting-started/common-tasks/update-formation'
             },
-            {
-              name: 'Releases',
-              href: '/getting-started/common-tasks/releases',
-            },
-            {
-              name: 'Reviewing feature branches',
-              href: '/getting-started/common-tasks/reviewing-feature-branches',
-            },
-            {
-              name: 'Documentation',
-              items: [
-                {
-                  name: 'Adding new pages to this site',
-                  href: '/getting-started/common-tasks/doc-page',
-                },
-                {
-                  name: 'Adding pages from Github to this site',
-                  href: '/getting-started/common-tasks/external-doc-page',
-                },
-                {
-                  name: 'Add Graphs to MDX',
-                  href: '/getting-started/common-tasks/add-graphs-to-mdx',
-                },
-                {
-                  name: 'Creating A Gatsby Plugin',
-                  href: '/getting-started/common-tasks/creating-gatsby-plugins',
-                },
-              ],
-            },
           ],
         },
-        {
-          name: 'Standards and conventions',
```

Comment:
Also deleted this. I don't know what we have planned for this section but I don't know if it falls under "getting started." Might be better under architecture.

Reply:
Initially, my idea was that the getting started section would be all the basic information you need to get started writing code: env setup, workflow, common tasks, and basic coding standards/conventions. I don't have a problem with moving, it might more sense in architecture.

Reply:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18142